postgres-appliance/spilok8s.yaml

```diff
@@ -1,41 +1,25 @@
-
 apiVersion: apps/v1alpha1
 kind: PetSet
 metadata:
+  name: &cluster_name spilodemo
   labels:
-    app: spilodemo
-  name: spilodemo
+    app: spilo
+    spilo-cluster: *cluster_name
 spec:
   replicas: 3
-  serviceName: spilodemo
+  serviceName: *cluster_name
   template:
     metadata:
       labels:
-        app: spilodemo
+        app: spilo
+        spilo-cluster: *cluster_name
       annotations:
         pod.alpha.kubernetes.io/initialized: "true"
     spec:
       containers:
-      - env:
-        - name: PGPASSWORD_SUPERUSER
-          value: tea
```

Comment:
This line was actually removed and  `tea` was not a placeholder, but the real password :)